packages/nocodb/src/lib/meta/helpers/webhookHelpers.ts

```diff
@@ -72,6 +72,26 @@ export async function validateCondition(filters: Filter[], data: any) {
       case 'notnull':
         res = data[field] !== null;
         break;
+      case 'allof':
+        res = (filter.value?.split(',').map((item) => item.trim()) ?? []).every((item) =>
+          (data[field]?.split(',') ?? []).includes(item)
+        );
+        break;
+      case 'anyof':
+        res = (filter.value?.split(',').map((item) => item.trim()) ?? []).some((item) =>
+          (data[field]?.split(',') ?? []).includes(item)
+        );
+        break;
+      case 'nallof':
+        res = !(filter.value?.split(',').map((item) => item.trim()) ?? []).every((item) =>
+          (data[field]?.split(',') ?? []).includes(item)
+        );
+        break;
+      case 'nanyof':
+        res = !(filter.value?.split(',').map((item) => item.trim()) ?? []).some((item) =>
+          (data[field]?.split(',') ?? []).includes(item)
+        );
+        break;
```

Comment:
may move (data[field]?.split(',') ?? []) out of .map since the value won't be changed in each iteration.